config.rb

```diff
@@ -28,20 +25,19 @@
 
 # Google analytics
 # Analytics ID must be inside the single quotes as it’s string
-#configure :development do
-#  set :analytics, ""
+configure :development do
+  set :analytics, ""
 #   set :analytics, "'UA-XXXXXXXX-X'"
-#end
+end
 
 # Insert your Google Analytics ID below
-# configure :build do
-#   set :analytics, "'UA-XXXXXXXX-X'"
-# end
+configure :build do
+  set :analytics, "'UA-XXXXXXXX-X'"
```

Comment:
Are we going to use these two things now? If no maybe leave them commented out

Reply:
Updated.